2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
Found it: http://stackoverflow.com/questions/9071990/are-pointers-always-set-to-nil-on-declaration
This works in ARC. Btw why you don't use ARC across the all sources?